### PR TITLE
TASK-2524 - implement draggable modal

### PR DESCRIPTION
**Pull request by Rodielm**
This PR added new functionality to dialog modal

### CODE REVIEW

File: src/webcomponents/variant/interpretation/variant-interpreter-grid.js [hunk 1]
```diff
@@ -1409,11 +1413,26 @@ export default class VariantInterpreterGrid extends LitElement {
...
+        $("#" + elementId).modal({
+            backdrop: false,
+            show: true,
+        });
+
+        $(`div[id='${elementId}']>div[class*='modal-dialog']`).draggable({
+            handle: ".modal-header"
+        });
+    }
+
```
Comment (jmjuanes) [inline src/webcomponents/variant/interpretation/variant-interpreter-grid.js:1435]: @Rodielm why do you need this function?

**Review by jmjuanes — CHANGES_REQUESTED** [review]

File: src/webcomponents/variant/interpretation/variant-interpreter-grid.js (hunk 1 above)
Comment (Rodielm) [reply src/webcomponents/variant/interpretation/variant-interpreter-grid.js:1435]: I'll remove it, it was another solution with jquery.

File: src/core/utils-new.js [hunk 2]
```diff
@@ -188,6 +188,45 @@ export default class UtilsNew {
...
+
+        self.addEventListener("mousemove", e => {
+            e.preventDefault();
+            if (isDown) {
+                mousePosition = {
+                    x: e.clientX,
+                    y: e.clientY
+                };
+                modalDialog.style.left = (mousePosition.x + offset[0]) + "px";
+                modalDialog.style.top = (mousePosition.y + offset[1]) + "px";
```
Comment (jmjuanes) [inline src/core/utils-new.js:225]: @Rodielm why do you need the `mousePosition` variable there? I think that you can just use the `e.clientX` and `e.clientY` values:

```js
modalDialog.style.left = (e.clientX + offset[0]) + "px";
modalDialog.style.top = (e.clientY + offset[1]) + "px";
```

File: src/core/utils-new.js [hunk 3]
```diff
@@ -188,6 +188,45 @@ export default class UtilsNew {
...
+        let isDown = false;
+        const modalDialog = modalElm.querySelector(".modal-dialog");
+        const modalHeader = modalElm.querySelector(".modal-header");
+        if (modalDialog) {
+            modalDialog.style.margin = "0";
+            modalDialog.style.left = (window.innerWidth * 0.30) + "px";
+            modalDialog.style.top = (window.innerHeight * 0.05) + "px";
+        }
+
+        modalHeader.addEventListener("mouseover", e => e.currentTarget.style.cursor = "move");
```
Comment (jmjuanes) [inline src/core/utils-new.js:203]: @Rodielm Sorry, but this is completely useless, as you can just set the cursor style without adding any event listener:

```js
modalHeader.style.cursor = "move";
```

The `cursor` style property is only applied when the pointer is over the element: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

**Review by jmjuanes — CHANGES_REQUESTED** [review]

**Review by jmjuanes — APPROVED** [review]

**Pull request merged**